WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -130,6 +158,25 @@ final public class InteractiveNotificationsManager : NSObject
     }
 
 
+    /// Replies to a comment
+    ///
+    /// - Parameters:
+    ///     - commentID: The comment identifier
+    ///     - siteID: The site identifier
+    ///     - content: The text for the comment reply
+    ///
+    private func replyToCommentWithCommentID(commentID: NSNumber, siteID: NSNumber, content: String) {
+        let context = ContextManager.sharedInstance().newDerivedContext()
+        let service = CommentService(managedObjectContext: context)
+
+        service.replyToCommentWithID(commentID, siteID: siteID, content: content, success: {
```

Comment:
I *think* we need a `performBlock` call here. Reason: it's a derived context.

Nevertheless, i think we're good anyways, since this particular service method doesn't really use the Core Data stack.